build/test-framework/run_tests.sh

```diff
@@ -12,7 +12,7 @@ sed -i '/^tsflags=.*/a ip_resolve=4' /etc/yum.conf
 yum install -y epel-release
```

Comment:
yum should become dnf

Reply:
Yea this is what was in here before, but I think we can make that adjustment here too.

Reply:
See #99